packages/icons/icons.json

```diff
@@ -4695,5 +4695,19 @@
         "tags": "folder, space, file-system, collaboration",
         "group": "interface",
         "codepoint": 62367
+    },
+    {
+        "displayName": "Code library",
+        "iconName": "code-library",
```

Comment:
We should rename this for consistency with some of our other "document" icons.

**I'd suggest renaming this to:** `document-code`

Using the "document" prefix matches up with other icons that already exist like: `document`, `document-open`, and `document-share`. It also matches the `"document-locked"` icon included with this PR.